var/spack/repos/builtin/packages/crtm/package.py

```diff
@@ -41,5 +41,8 @@ class Crtm(CMakePackage):
     version("2.4.0", commit="a831626")
     # Uses the tip of REL-2.3.0_emc branch
     version("2.3.0", commit="99760e6")
```

Comment:
Assuming these version values are tags or should be consistent with tags, this should be "v2.4.0_emc" and "v2.3.0_emc"

`v2.4.0_emc` tag has the commit: "5ddd0d6"
`v2.3.0_emc` tag has the commit: "99760e6"

`v2.3-jedi.4` is correct

`v2.4_jedi` should be "0ee3593"

Not sure why there's a discrepancy between internal and public repos on the commit hashes for the tags. I'll investigate, but for now, let's use what's on public only.

Let me know if I'm misunderstanding something.